app/config/AppConfig.scala

```diff
@@ -105,6 +105,13 @@ class AppConfig @Inject()(servicesConfig: ServicesConfig, configuration: Configu
     "cy" -> Lang("cy")
   )
 
+  private def traderKnownFactsReferenceDataService: String =
+    if (isEnabled(StubGetTraderKnownFacts)) {
+      servicesConfig.baseUrl("emcs-tfe-reference-data-stub")
```

Comment:
[Question] Why is this doing a check here as the `emcs-tfe-reference-data` service can just be set to use the stub vs oracle.

Reply:
Are you asking why we're using a feature switch?

Reply:
@ericohwotu the existing feature switch on emcs-tfe-reference-data is an all-or-nothing thing - either it always uses Oracle or it always uses the stub. This feature switch means that we can have it use the stub specifically for trader-known-facts whilst using Oracle for reference-data, for example in Staging/QA.
We have two options really:

- Add an extra feature switch to emcs-tfe-reference-data so that we have two in there, one for trader-known-facts and the other for everything else (extra tech debt story is needed to take this feature switch out of the rest of the frontends in this case)
- Leave this as-is

Up to you which we go for, I don't really mind either way, but I don't think it should be addressed in this PR as it's common across all frontends.

Reply:
Think point 1 will save confusion in the future.

Reply:
I agree, I'll raise a tech debt ticket.